pkg/odo/cli/component/devfile.go

```diff
@@ -152,3 +153,22 @@ func (do *DeleteOptions) DevfileComponentDelete() error {
 	log.Successf("Successfully deleted component")
 	return nil
 }
+
+func validateURL(url []envinfo.EnvInfoURL) {
+	// validateURL validates if env.yaml contains a valide URL for the current pushtarget
+	// display a warning if no url(s) found for the current push target, but found url(s) for another push target
+	dockerURLExists := false
+	kubeURLExists := false
+	for _, element := range url {
+		if element.Kind == envinfo.DOCKER {
+			dockerURLExists = true
+		} else {
+			kubeURLExists = true
+		}
+	}
+	if pushtarget.IsPushTargetDocker() && !dockerURLExists && kubeURLExists {
+		log.Warningf("found url(s) defined for ingress/route, but no valid urls with exposed port defined")
```

Comment:
Should be capitalized.

Reply:
url needs to be capitalized too. 

This may be a better output: `Found URL(s) defined for ingress/route, but no valid URLs with exposed port defined`

Reply:
`Found URL(s) defined for Kubernetes, but no valid URLs for Docker`?

Reply:
updated